### ISSUE TITLE
Plot figures with cartopy

### ISSUE THREAD

**Issue by leticiafabre** [oceanhackweek/ohw22-proj-gcmfilters#7]
Just a reminder to myself that I would like to try using cartopy to make cool figures (if there is time) 😄

**Comment by anaclarabmariani** [comment]
I managed to plot it, but I'm still having some problems... The values for the plotted scalar variables didn't seem right (The code is in my fork: https://github.com/anaclarabmariani/ohw22-proj-gcmfilters/blob/main/Exploring_Data_NCAR.ipynb)

![filtered_SSH_test_cartopy](https://user-images.githubusercontent.com/81709935/185444931-e35764ab-ac34-4824-b5e5-2632f5a89de0.png)

**Comment by NoraLoose** [comment]
@anaclarabmariani would you like to submit a pull request (PR) with this notebook? Then I can leave comments on the notebook, and we can work on fixing the issue, before merging the PR. You can always make changes after the PR is submitted, as long as the PR is not merged.

**Comment by anaclarabmariani** [comment]
Sure! Good idea! Let me try to do that.

**Comment by paigem** [comment]
@anaclarabmariani I have taken a quick look tonight at your notebook to figure out the cartopy plotting weirdness, and I don't have any answers off the top of my head for why the fields don't appear correctly. I'll dig more into it tomorrow morning! 😊

**Comment by NoraLoose** [comment]
Yes, I was going to write the same thing as @paigem :smile: It may have to do with the masking. Let's definitely work more on this tomorrow. I want to figure it out before the project presentations! 🙌

**Comment by paigem** [comment]
@anaclarabmariani I figured out how we can make your regional plots look nice with cartopy! 😅 It's not the most elegant way to do it, but I used a work-around and got this nice figure:  

![image](https://user-images.githubusercontent.com/26591824/185646339-57b26afa-b34a-4fb0-8509-f5374ad68c0f.png)

I'm putting together a notebook with all of my steps and will add it to this repo as soon as I'm done!
